geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1144,7 +1144,10 @@
    * <U>Allowed values</U>: true or false
    * </p>
    * <U>Since</U>: GemFire 8.0
+   *
+   * @deprecated Since Geode 1.6 use "import cluster-configuration" command instead
    */
+  @Deprecated
```

Comment:
Regular users will likely never notice this property is deprecated as they rarely check the source code. 
The documentation is [already updated ](https://github.com/apache/geode/blob/develop/geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb#L393), though, but maybe it's a good idea to log a warning as part of the `start locator` command when the property is actually used?.

Reply:
the gfsh help message  contains the "deprecated" explanation since 1.6. Bug logging a warning seems a good idea, but not the scope of this PR.